src/test/java/com/neuvector/ScannerTest.java

```diff
@@ -45,14 +57,95 @@ public void scanRegistryTest(){
         String repositoryTag = "3.6";
         String nvScannerImage = "neuvector/scanner:latest";
         String nvRegistryURL = registryURL;
-        String nvRegistryUser = "xxx";
-        String nvRegistryPassword = "xxx";
+        String nvRegistryUser = "";
+        String nvRegistryPassword = "";
         String mountPath = "/temp";  //mountPath is an optional parameter. It will use "/var/neuvector" by default.
 
         Registry registry = new Registry(registryURL, regUser, regPassword,repository,repositoryTag);
         NVScanner scanner = new NVScanner(nvScannerImage, nvRegistryURL, nvRegistryUser, nvRegistryPassword, mountPath);
 
         ScanRepoReportData scanReportData = Scanner.scanRegistry(registry, scanner, license);
+
+        File file = new File(mountPath + "/scan_result.json");
+        UserPrincipal user = getUserPrincipal(mountPath, file);
+
+        assertTrue(!user.getName().equals("root") || !user.getName().contains(System.getProperty("user.name")));
         assertTrue( scanReportData != null );
     }
+
+    @Test
+    public void scanRegistryTestWhenFileIsCreatedByNoRootUser() throws IOException {
+        String license = "r+wzvemF4mHMIBkvKHXTuuLzw4ML/RW2ARfbC06Qem1VdQ8DXcMWNKbd8GD0uYiLoS9d9XPM8bff4ag66fqwdOa42oNXk9dzP1TTRg0zvVKJNMxbxKAuGMY24x0NlXstc3Jl2BcXq7aTHrlQwzJ/7qD8157+YfcjKJbywbt3WvvdpuoJtJIuKaO9hOfGu9/zbRZznEtZiJj56xaRMVZ/ZkPHIMseRWML7FUr1oft20PYU7ltf1BGPT/LlocAXAJucBpUz5iosRzS1iYJhU4M1kdBhEwQHl6FZXrtzJGYpM21587FyYrkvoIPQNlIijaAOiCXR060PJNfcQzOzEpYRidhVPnHIHQuIn8gcYuU3XwPB9Pq3tGzERpr0aB64EjSSanpcIh/XEhnXijiRP6CfxsHx8RvD/jkXy5VlJpTVFh1Iu7ywE6fePHb3hIe16uCtdYB8fu2p8ksCcnxXbjBUkKvdLLxq5gvOCfCJxknd4GdIS3UJySNblgmlfU6liYcgt555TitN5DAjuMhTaoxj9FoPPwJsl4rrf2+32cqUFI+9+UdNsSd2gWtS/8T2ifPHNVMA7fdPdIWx+/4jXP8u484cLqGG9RutNGPo0k7cmxmgx29FvVOMvpkotD8IZsbCfl8sFHtFRP7PegWYPghZf6yUkw5m4SD+bacVZalnLizZKbDgFuzPmH1LgrroVN1Ngf06JVfO6lDodbAOactsJC7P8DOPRUoOtnCf3ffgshPh+QSTKDOmNvFU51VB7MoLlZiKNaVP6yMqS1bUwxRpI5eehqhqCHy46PWfRHF2EnbiQOyYIjJSY5jE54TzOaKZ613GgdIaMN+mbXg+UWajDevwCvKQo+CrwpBoX72tP/as9G3zIJILaKuXbQRvTtZbgGREMSoLIDxPBYe3FVqMHFg/jUE9iPbsF16JHv/z3EzcAYKwYSyDEyVppTvLjQY0t/Vul4SEj+QWpyW7elAUmatzHdAcX2SB3oDOhUtDyIxw8rtenD9hCOfspJtEFknhtM+Zm1WuoQYRM5/paEVvbHZnu2XHY2cnBtxJZAD8wKktnSPGg3foCXQpQcSEyhfWhVT14WX2/GxgbdwIuxSBcoHlcduca5tKtlKAhJ4kjPPZeQc5A18z48pLRU=";
```

Comment:
Value of the license is not needed by neuvector container.. It can be replaced by an empty string. If you ran one of the existing tests successfully, you can verify this. First you can make sure an existing test already runs (without errors).. Then you can modify one of them as I do here to clear the `license` value and verify it still runs successfully (again no errors).. I am attaching a sample run from my environment:

<img width="1286" alt="Screen Shot 2023-03-02 at 2 03 20 PM" src="https://user-images.githubusercontent.com/1974123/222526554-0be9d88c-d94e-4726-a941-22523294e63e.png">

Reply:
You are right, I deleted the license: f7a254ce9deebdd6914e0f93e004f721b569cd77